src/api/groups.controller.ts

```diff
@@ -49,6 +49,13 @@ export class GroupsController {
     return session.getGroup(id);
   }
 
+  @Put(':id/groupAdminOnly')
+  @SessionApiParam
+  @ApiOperation({ summary: 'Set group admin only.' })
+  setGroupAdminOnly(@SessionParam session: WhatsappSession, @Param('id') id: string) {
+    return session.setInfoAdminsOnly(id);
```

Comment:
Let's set it by default to `true`, so we can extend it later to allow disable that setting too
```suggestion
    return session.setInfoAdminsOnly(id, true);
```